packages/cln_plugin/example/cln_plugin_class_example.dart

```diff
@@ -29,6 +29,7 @@ class MyPlugin extends Plugin {
         usage: "",
         description: "This is an example of custom rpc method",
         callback: (plugin, request) => firstMethod(plugin, request));
+    log(level: "debug", message: "messageString");
```

Comment:
```suggestion
    log(level: "info", message: "messageString");
```

Try info :) not sure if exist `debug`